index.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable import/no-cycle */
+import Book from './modules/books.js';
+import displayToPage from './modules/desplayToPage.js';
+/* eslint-disable import/no-cycle */
+import { bookListHandler, addSectionHandler, contactInfoHandler } from './modules/singlePage.js';
+import { DateTime } from './modules/luxon.js';
+
+const linForBookList = document.querySelector('.for_list_link');
+const linkForAddSection = document.querySelector('.for_addSection_link');
+const linkForContInfo = document.querySelector('.for_contact_link');
+
+const addButton = document.querySelector('#add');
+const titleInput = document.querySelector('#title');
+const autherInput = document.querySelector('#author');
+/* eslint-disable import/no-mutable-exports,import/prefer-default-export */
```

Comment:
- [ ] it would be better as the previous reviewers asked you should remove the disable linter lines from all the files in order to keep the code clean and pro